master/internal/db/postgres_rbac_intg_test.go

```diff
@@ -546,10 +622,9 @@ func TestPermissionMatch(t *testing.T) {
 		require.NoError(t, err, "error when searching for permissions")
 		require.Equal(t, noWorkspaces, workspaces)
 
-		workspaces, err = GetNonGlobalWorkspacesWithPermission(ctx, userIDEditorRestricted,
+		_, err = GetNonGlobalWorkspacesWithPermission(ctx, userIDEditorRestricted,
```

Comment:
How come we deleted `workspaces` here?

Reply:
Linter was complaining because it is an ineffectual assignment

Reply:
Was it because [this](https://github.com/determined-ai/determined/pull/9822/files#diff-c196aa831ff08c80261da43db7dd824f4bdb961369d16cfb67e17e61ff29c0f8L552) line was deleted?

Reply:
oh weird, idk how that got deleted but I'll fix it now!

Reply:
fixed!